plugin/ochttp/client.go

```diff
@@ -25,9 +25,7 @@ import (
 // Transport is an http.RoundTripper that instruments all outgoing requests with
 // OpenCensus stats and tracing.
 //
-// The zero value is intended to be a useful default, but for
-// now it's recommended that you explicitly set Propagation, since the default
-// for this may change.
+// Use NewTransport to create a Transport with the default configuration.
```

Comment:
I think this is a confusing choice. Zero-values are expected to support the default configuration.

We should deprecate the use of Transport zero value if this is the way to go.

Reply:
I would like to deprecate use of the zero value. Just not exactly sure how to go about this, except for what I have done here which is to document that NewTransport should be used.